alongtheway.txt

```diff
@@ -534,9 +534,9 @@ Sometimes the junk in our life is a result of reaping what we have sown but some
 
 Day 3: Mary's journey to Bethlehem
 Read: Luke 2:1-21 especially verses 4-7
-Try to keep yesterday's reading in mind. Look back at what you read and wrote down about what the angel Gabriel said about who Jesus would be.
+Try to keep yesterday's reading in mind. Look back at what you read and wrote concerning the angel Gabriel's words about who Jesus would be.
 
-Reflect: When I stop to think about it, and try to put myself in Mary’s position, this story is going pretty much exactly the opposite of how I would have thought things would go. If an angel appeared to me and told me I was highly favored and carrying the son of God, savior of the world, personally, I would expect to be birthing him somewhere other than in a barn, in a strange place, away from all of my family and friends.
+Reflect: When I stop to think about it, and try to put myself in Mary’s position, this story is going pretty much exactly the opposite of how I would have thought things would go. If an angel appeared to me and told me I was highly favored and carrying the son of God, savior of the world, personally, I would expect to be birthing him somewhere other than a strange place, in a barn, away from all of my family and friends.
```

Comment:
I agree - either works. I do like the cadence of the original, maybe with a period instead of a comma there? 

Sent from my iPhone

> On Jan 15, 2016, at 11:38 PM, Joshua Maxwell <notifications@github.com> wrote:
> 
> In alongtheway.txt:
> 
> >  
> > -Reflect: When I stop to think about it, and try to put myself in Mary’s position, this story is going pretty much exactly the opposite of how I would have thought things would go. If an angel appeared to me and told me I was highly favored and carrying the son of God, savior of the world, personally, I would expect to be birthing him somewhere other than in a barn, in a strange place, away from all of my family and friends.
> > +Reflect: When I stop to think about it, and try to put myself in Mary’s position, this story is going pretty much exactly the opposite of how I would have thought things would go. If an angel appeared to me and told me I was highly favored and carrying the son of God, savior of the world, personally, I would expect to be birthing him somewhere other than a strange place, in a barn, away from all of my family and friends.
> did you want the emphasis by repeating "in a___, in a _____" or does this reordering work? I could go with either...
> 
> —
> Reply to this email directly or view it on GitHub.
>